.evergreen.yml

```diff
@@ -296,6 +398,65 @@ tasks:
           AUTH_TEST: "true"
     - func: "attach-test-results"
 
+- name: integration-4.4
+  commands:
+    - func: "fetch source"
+    - command: expansions.update
+    - func: "download mongod"
+      vars:
+        mongo_version: "4.4"
+    - func: "start mongod"
+    - func: "wait for mongod to be ready"
+    - func: "run tool integration tests"
+      vars:
+        INT_TEST: "true"
+    - func: "attach-test-results"
+
+- name: integration-4.4-auth
+  commands:
+    - func: "fetch source"
+    # Concat auth args
+    - command: expansions.update
+      params:
+        updates:
+          - key: "mongod_args"
+            concat: " --auth"
+          - key: "auth_username"
+            value: "passwordIsTaco"
+          - key: "auth_password"
+            value: "Taco"
```

Comment:
Can the username and password be kept in evergreen profile? This is a low risk credential, but in the future, all credential related values should be stored in evergreen project profile, since once it's committed to github, it cannot be removed.